packages/wallet/src/redux/sagas/adjudicator-watcher.ts

```diff
@@ -64,6 +64,9 @@ function* dispatchEventAction(event: AdjudicatorEvent) {
         })
       );
       break;
+    case AdjudicatorEventType.Concluded:
+      yield put(actions.concludedEvent({channelId}));
+      break;
```

Comment:
Shouldn't there be a `default:` here?

Reply:
There should be - what's an appropriate default for this switch block?